src/libfetchers/git.cc

```diff
@@ -290,13 +305,17 @@ struct GitInputScheme : InputScheme
         if (maybeGetStrAttr(attrs, "type") != "git") return {};
 
         for (auto & [name, value] : attrs)
-            if (name != "type" && name != "url" && name != "ref" && name != "rev" && name != "shallow" && name != "submodules" && name != "lastModified" && name != "revCount" && name != "narHash" && name != "allRefs" && name != "name" && name != "dirtyRev" && name != "dirtyShortRev")
+            if (name != "type" && name != "url" && name != "ref" && name != "rev" && name != "shallow"
```

Comment:
Maybe arrange it to be one symbol at a line? This is already accumulating handling friction...